Identity/IdentityFactory.swift

```diff
@@ -11,8 +11,11 @@ import Foundation
 public struct IdentityFactory {
     static func makeModule(
         analyticsLogger: Analytics,
-        identityPersister: IdentityPersisting
+        identityPersister: IdentityPersisting,
+        graphQLManager: GraphQLManageable
     ) -> Identity {
-        return IdentityImplementing(analyticsLogger: analyticsLogger, identityPersister: identityPersister)
+        return IdentityImplementing(analyticsLogger: analyticsLogger,
+                                    identityPersister: identityPersister,
+                                    sso: SSORepository(graphQLManager: graphQLManager))
```

Comment:
Should the Factory create the SSORepository or be passed the SSORepository? What pattern do we use normally?

Reply:
The factory should create `SSORepository`. The same approach is used in `SellFactory`

Reply:
Thanks